rotkehlchen/api/rest.py

```diff
@@ -4351,3 +4367,14 @@ def refresh_general_cache(self) -> dict[str, Any]:
                 status_code=HTTPStatus.CONFLICT,
             )
         return OK_RESULT
+
+    def get_eth_airdrops_metadata(self) -> Response:
+        """Returns a list of airdrops on ETH."""
+        result = []
+        for identifier, (_, _, _, name, _) in AIRDROPS.items():
+            result.append({
+                'identifier': identifier,
+                'name': name,
+                'icon': f'/protocols/icon?protocol={identifier}',
```

Comment:
In the case of the protocols we just provide the name of the file. You can check for eaxmple https://github.com/rotki/rotki/blob/1169de3db17ef203a34d87a5fa49b8e54264ffe9/rotkehlchen/chain/ethereum/modules/aave/v2/decoder.py#L333 in the case of the decoders we collet them later by calling this function and then return the structure. In your case just add the file name here